libchisel/src/deployer.rs

```diff
@@ -25,6 +25,23 @@ impl<'a> ChiselModule<'a> for Deployer {
     }
 }
 
+use std::collections::HashMap;
+
+impl ModuleConfig for Deployer {
+    fn with_defaults() -> Result<Self, ModuleError> {
+        Err(ModuleError::NotSupported)
+    }
+
+    fn with_config(config: &HashMap<String, String>) -> Result<Self, ModuleError> {
+        let preset = config.get("preset");
+        if preset.is_none() {
```

Comment:
This expression can be simplified:
```
if let Some(preset) = config.get("preset") {
    Deployer::with_preset(preset)
} else {
    Err(ModuleError::NotSupported)
}
```

Reply:
Yeah now it can, before we had #171 this wasn't as simple.